mmdet/apis/export.py

```diff
@@ -55,6 +55,7 @@ def patch_nncf_model_for_alt_ssd_export(model):
 def export_to_onnx(model,
                    data,
                    export_name,
+                   output_names,
```

Comment:
Do you pass in here anything apart from `['boxes', 'labels']`? If not, why do you need this extra argument?

Reply:
Actually, you're right, we don't need it and probably it provokes the problem with CI. I'll remove it